var/spack/repos/builtin/packages/vtk-h/package.py

```diff
@@ -63,9 +63,8 @@ class VtkH(Package, CudaPackage):
     variant("logging", default=False, description="Build vtk-h with logging enabled")
     variant("contourtree", default=False, description="Enable contour tree support")
 
-    # use cmake 3.14, newest that provides proper cuda support
-    # and we have seen errors with cuda in 3.15
-    depends_on("cmake@3.14.1:3.14.99", type='build')
+    # Certain CMake versions have been found to break for our use cases
+    depends_on("cmake@3.14.1:3.14.99,3.18.2", type='build')
```

Comment:
overall looks good, but should this one also be `3.18.2:`

Reply:
Good catch